docker/importer/importer.py

```diff
@@ -598,6 +802,15 @@ def main():
       '--strict_validation',
       help='Fail to import entries that does not pass validation',
       default=False)
+  parser.add_argument(
+      '--delete',
+      help='Propagate record deletions from source to Datastore',
```

Comment:
Should we clarify that this will skip the usual import mode? i.e. this is running in a deletion only mode?

Reply:
Clarified in 94f2e87